cli_integration_tests/inputs/Cas9.amplicons.genome.txt

```diff
@@ -0,0 +1,3 @@
+FANC	CGGATGTTCCAATCAGTACGCAGAGAGTCGCCGTCTCCAAGGTGAAAGCGGAAGTAGGGCCTTCGCGCACCTCATGGAATCCCTTCTGCAGCACCTGGATCGCTTTTCCGAGCTTCTGGCGGTCTCAAGCACTACCTACGTCAGCACCTGGGACCCCGCCACCGTGCGCCGGGCCTTGCAGTGGGCGCGCTACCTGCGCCACATCCATCGGCGCTTTGGTCGG	GGAATCCCTTCTGCAGCACC
+HEK3	GGAAACGCCCATGCAATTAGTCTATTTCTGCTGCAAGTAAGCATGCATTTGTAGGCTTGATGCTTTTTTTCTGCTTCTCCAGCCCTGGCCTGGGTCAATCCTTGGGGCCCAGACTGAGCACGTGATGGCAGAGGAAAGGAAGCCCTGCTTCCTCCAGAGGGCGTCGCAGGACAGCTTTTCCTAGACAGGGGCTAGTATGTGCAGCTCCTGCACCGGGATACTGGTTGACAAG	GGCCCAGACTGAGCACGTGA
+GENOME	CACCAGGAGAGGAGGGACCTGGCCCTTCAGGGTCGAGCTCAACAGAGGAAAAGATCTCAGGGCACCCAGAGCCCAGTGGCTTTCAGCACCTGCATGAAAATCAGAGATCAACCAGATTACCCCATATTGCCCAAGAGAAAACTGAGGCCAGAAAGTGATGGAGCTTGCCCAGGACCCCAGAGGGTGTTAGAGGCAGGCCAGAACCAGAGGACAGCTCTGTCCCAGC	ATCAGAGATCAACCAGATTA
```

Comment:
I think by adding this here there is an additional amplicon, which is not testing grabbing the amplicon from the genome. I would suggest removing this, and making sure that the reads you added are being picked up properly from the genome. The goal of the genome (and, in part, mixed) mode is that you don't have to specify the amplicon sequence in this way, that the amplicon sequence will be picked up from the genomic sequence.

Reply:
This is wrong! You do need the additional amplicon in this file.